runtime/src/main/java/io/quarkiverse/cxf/transport/CxfHandler.java

```diff
@@ -251,6 +263,32 @@ private String getBaseURL(HttpServerRequest request) {
         return reqPrefix;
     }
 
+    private HttpServletRequest checkXForwardedHeaders(HttpServletRequest request) {
```

Comment:
Might make sense to reference https://github.com/apache/cxf/blob/cxf-3.4.5/rt/transports/http/src/main/java/org/apache/cxf/transport/servlet/AbstractHTTPServlet.java#L301

Reply:
Excellent point - I added a brief comment as well that says the method is leveraging the [Quarkus HTTP proxy configuration properties](https://quarkus.io/guides/http-reference#reverse-proxy), instead of purely HTTP headers.  Will need to get all added to the docs.